client/src/subapps_key_concept_panels/FAQPanel.js

```diff
@@ -0,0 +1,30 @@
+import _ from "lodash";
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
+
+import {
+  create_text_maker_component,
+  KeyConceptList,
+} from "src/components/index";
+
+import est_lang from "src/EstimatesComparison/estimates_comparison_questions.yaml";
+
+const { TM } = create_text_maker_component([est_lang]);
+
+export class FAQPanel extends React.Component {
```

Comment:
Why not use the key concept panel? It should be to reuse the `curried_render` in key_concept_panel